### ISSUE TITLE
User agent detection when content downloaded or streamed from web browser

### ISSUE THREAD

**Issue by theboubougne** [skymethod/livewire-web#1]
Some podcast services are adding a ?from=app_name" parameter to the urls to be identified 
Maybe you can check for this in your reflections podcast feed.
Thanks!

**Comment by johnspurlock-skymethod** [comment]
Good idea.  Can you give me some real life app name slugs that are actually in use today that you want to be included?  Is there a master list somewhere?

I found a list in the opawg / [podcast-rss-useragents](https://github.com/opawg/podcast-rss-useragents) repo, but I'm unclear if anyone actually uses it yet.

**Comment by theboubougne** [comment]
I think the name of the apps with no space and case insensitive should be
used: pocketcasts, overcast, podfriend, podcastaddict, playerfm, ...

**Comment by gsimolke1** [comment]
Hi John, 

So nice to meet you! I am writing to seek approval to use a data point from  in an upcoming presentation Hala Taha is conducting at the IAB Upfronts on 5/9 in NYC. 

Here are the screenshots of the slides where the data points appear, as well as the citation. Let us know if you would like any edits and if we are approved to use this.

Source:



2 Spurlock, J. (2024, February 1). Tracking the Impact of the Auto-Downloading Changes to Apple Podcasts in IOS 17. [livewire.io](http://livewire.io/): Podcasting ▷ Forward.


Appreciate any help or support you can provide!
<img width="855" alt="Screenshot 2024-05-04 at 8 53 19 PM" src="https://github.com/skymethod/livewire-web/assets/168949189/847c7623-6307-4c6a-8556-a12eef84130d">

**Comment by johnspurlock-skymethod** [comment]
Thanks for the citation @gsimolke1, approved!
